hpOneView/connection.py

```diff
@@ -359,6 +359,10 @@ def __do_rest_call(self, http_method, uri, body, custom_headers):
                 task = body
 
             return task, body
+
+        if 'category' in body and body['category'] == 'tasks':
```

Comment:
This change allows the Python SDK to know when a task resource is returned in the response body when the response status is not 202.  Specifically, VCM Migration Manager does not return a response status of 202 but it does return a task resource for POST and PUT.